opensrp-chw/src/androidTest/java/org/smartregister/chw/activity/ba/RemoveMemberTestsBa.java

```diff
@@ -0,0 +1,242 @@
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring("Remove existing family member"))
+                .perform(click());
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring(Configs.AdditionalTestDataHelper.memberTwoFirstname
+                + " " + Configs.AdditionalTestDataHelper.memberTwoSecondname + " "
+                + Configs.TestConfigHelper.familyName
+                + ", " + Configs.AdditionalTestDataHelper.extraMemberAge2))
+
+                .perform(click());
+        Activity activity = getCurrentActivity();
+        onView(withId(getViewId((JsonFormActivity) activity, "step1:remove_reason")))
+                .perform(click());
+        onView(ViewMatchers.withSubstring("Moved away"))
+                .perform(click());
+        onView(withId(getViewId((JsonFormActivity) activity, "step1:date_moved")))
+                .perform(click());
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring("done")).perform(click());
+        Thread.sleep(500);
+        onView(withId(R.id.action_save))
+                .perform(click());
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring("REMOVE"))
+                .check(matches(isDisplayed()));
+    }
+
+    @Test
+    @Order(order = 5)
+    public void removeFamilyMemberWithDeathAsReason() throws Throwable {
+        onView(withHint("Search name or ID"))
+                .perform(typeText(Configs.TestConfigHelper.familyName), closeSoftKeyboard());
+        onView(ViewMatchers.withSubstring(Configs.TestConfigHelper.familyName + " Family"))
+                .perform(click());
+        Thread.sleep(500);
+        utils.openFamilyMenu();
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring("Remove existing family member"))
+                .perform(click());
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring(Configs.TestConfigHelper.kidNameFirst
+                + " " + Configs.TestConfigHelper.kidNameSecond + " " + Configs.TestConfigHelper.familyName + ", 0"))
+                .perform(click());
+        Activity activity = getCurrentActivity();
+        onView(withId(getViewId((JsonFormActivity) activity, "step1:remove_reason")))
+                .perform(click());
+        onView(ViewMatchers.withSubstring("Died"))
+                .perform(click());
+        Thread.sleep(500);
+        onView(withId(getViewId((JsonFormActivity) activity, "step1:date_died")))
+                .perform(click());
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring("done")).perform(click());
+        Thread.sleep(500);
+        onView(withId(R.id.action_save))
+                .perform(click());
+        Thread.sleep(500);
+        onView(ViewMatchers.withSubstring("REMOVE"))
+                .perform(click());
+        Thread.sleep(500);
+    }
+
+    @Test
+    @Order(order = 1)
+    public void removeFamilyMemberWithoutReason() throws InterruptedException{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426621)